packages/package_info/example/lib/main.dart

```diff
@@ -72,7 +92,10 @@ class _MyHomePageState extends State<MyHomePage> {
       appBar: new AppBar(
         title: const Text('Plugin example app'),
       ),
-      body: new Center(child: new Text('App version is: $_version\n')),
+      body: new Center(
```

Comment:
Printing multiple lines of text line this doesn't secure proper alignment. Please change to something like this:

```dart
      body: new Center(
        child: new Row(
          children: <Widget>[
            new Text(text1),
            new Text(text2),
            new Text(text3),
          ],
        ),
      ),
```

Reply:
OK, I change back. 
Package_name of the example is too long, so ...